apps/dashboard/src/app/event/page.tsx

```diff
@@ -1,17 +1,51 @@
 "use client"
 
-import type { Event } from "@dotkomonline/types"
-import { formatDate } from "@dotkomonline/utils"
+import type { Event, EventType } from "@dotkomonline/types"
 import { Icon } from "@iconify/react"
-import { Anchor, Box, Button, ButtonGroup, Skeleton, Stack } from "@mantine/core"
+import {
+  ActionIcon,
+  ActionIconGroup,
+  Anchor,
+  Button,
+  Group,
+  Skeleton,
+  Stack,
+  Text,
+  TextInput,
+  Title,
+  Tooltip,
+} from "@mantine/core"
 import { createColumnHelper, getCoreRowModel, useReactTable } from "@tanstack/react-table"
 import Link from "next/link"
 import { useMemo } from "react"
 
 import { GenericTable } from "@/components/GenericTable"
+import { formatDate } from "date-fns"
+import { nb } from "date-fns/locale"
```

Comment:
We should set locale options for date-fns in the root layout.tsx instead

Reply:
Can do this in another PR